### PR TITLE
revoke device

### PR DESCRIPTION
**Pull request by ElijahGeorge**
If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?
Prevents Notifications from being sent to signed out device
#### Related JIRA tickets:
https://jira.amida.com/browse/ORANGE-1029 
#### How should this be manually tested?
Assert you do not receive any Push notifications after signing out from a device
#### Screenshots (if appropriate):

### CODE REVIEW

**Comment by ElijahGeorge** [comment]
@dbrezack. Good point. But this only get's called when the user signs out of a device hence only the sign-out action by the device user can revoke a token. Also, while the probability of different users having the same token will likely be very low, we are not exactly sure what Apple and Google's token rotation policy is. This might necessitate using the user to query for the token as well. Thoughts?

**Comment by ElijahGeorge** [comment]
@dbrezack It's also important to note that the token being revoked is not for any kind of access. It's simply what we send to the Apple or FCM servers to be used to dispatch a notification to a device

**Comment by dbrezack** [comment]
@ElijahGeorge it sounds like the token identifies the device.  Does the device have access to this token? If so we could revoke all tokens for the device upon login and then register their token & username with the notification service. That way you will receive notifications for the last account logged in until someone new signs in.

**Comment by ElijahGeorge** [comment]
Yes @dbrezack . The token is stored in AsyncStorage on the device. Also we are already persisting a new token to the Notification MS when the user logs in (which may or may not be the same as the previous one), hence we do a findOrCreate with device tokens. I think Jon and Aarons concerns were that notifications were still being sent to the phone after logging out.

File: .env.example [hunk 1]
```diff
@@ -31,6 +31,7 @@ PUSH_NOTIFICATIONS_APN_ENV=development
 PUSH_NOTIFICATIONS_APN_TEAM_ID=
 PUSH_NOTIFICATIONS_APN_KEY_ID=
 PUSH_NOTIFICATIONS_APN_TOPIC=
+PUSH_NOTIFICATIONS_APN_KEY_PATH=/app/iosKey.p8
```
Comment (mountHouli) [inline .env.example]: Please document this new environment variable in `README.md` according to this format:

https://github.com/amida-tech/amida-notification-microservice#variable_name-required-if-it-actually-is-the-default-value

**Pull request merged**